visidata/guide.py

```diff
@@ -86,4 +95,42 @@ def iterload(self):
                 yield [i] + list(m.groups())
                 i += 1
 
-BaseSheet.addCommand('', 'open-guide', 'vd.push(GuideGuide("VisiData_Guide"))')
+    def openRow(self, row):
+        guidename = row[1]
+        return vd.getGuide(guidename)
+
+class GuideSheet(Sheet):
+    rowtype = 'lines'
+    filetype = 'guide'
+    columns = [
+            ItemColumn('linenum', 0, type=int, width=0),
+            ItemColumn('guide', 1, width=80, displayer='full'),
+            ]
+    precious = False
+    guide = ''
+
+    def iterload(self):
+        winWidth = 78
+        for startingLine, text in enumerate(self.guide.splitlines()):
+            text = text.strip()
+            if text:
+                for i, (L, _) in enumerate(wraptext(str(text), width=winWidth)):
+                    yield [startingLine+i+1, L]
+            else:
+                yield [startingLine+1, text]
+
+
+
+@VisiData.api
+def getGuide(vd, guidename):
+    if guidename in vd.guides:
+        return vd.guides[guidename]
+    vd.warning(f'there is no guide: {guidename}')
```

Comment:
We need to decide if we're minimizing words (`"no guide named {guidename}"`) or if we want proper technical documentation (`"The {guidename} guide has not yet been written. If you'd like to contribute, ..."`).  I vote the former for now, since that's what VisiData's style to date has been.